src/server.js

```diff
@@ -0,0 +1,53 @@
+import express from 'express';
+import { config } from 'dotenv';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import swaggerUI from 'swagger-ui-express';
+import docs from '../swagger.json';
+import indexRouter from './routers';
+
+config();
+
+/**
+ * Setup of the application server
+ * @function startServer
+ * @param {Number} port
+ * @return {Promise} Resolves to the HTTP server
+ */
+const startServer = port =>
+  new Promise((resolve, reject) => {
+    if (!port) {
+      reject(new Error('The server must be started with an available port'));
+    }
+
+    const app = express();
+    app.set('json spaces', 2);
```

Comment:
why do we need this?

Reply:
This will make the response easy to read. It will format the JSON response coming from the server using 2 spaces indentation.

Reply:
You mean coming from the client?

Reply:
Yea

Reply:
This makes sense...LGTM